magik-checks/src/test/java/nl/ramsolutions/sw/magik/checks/MagikIssueDisabledCheckerTest.java

```diff
@@ -12,72 +12,73 @@
 class MagikIssueDisabledCheckerTest {
 
   @Test
-  void testNotDisbled() throws ReflectiveOperationException {
-    final String code = "show(1)\n";
-    final MagikFile magikFile = new MagikFile(MagikFile.DEFAULT_URI, code);
-
-    final MagikCheckHolder holder =
-        new MagikCheckHolder(ForbiddenCallCheck.class, Collections.emptySet(), true);
-    final MagikCheck check = holder.createCheck();
-    final List<MagikIssue> issues = check.scanFileForIssues(magikFile);
-    final MagikIssue issue = issues.get(0);
-
-    final boolean issueDisabled = MagikIssueDisabledChecker.issueDisabled(magikFile, issue);
-    assertThat(issueDisabled).isFalse();
+  void testNotDisabled() throws ReflectiveOperationException {
+    assertIssueDisabled("show(1)\n", false);
```

Comment:
Call on `this`, to make it explicit/clear where the method is coming from. E.g., `this.assertIssueDisabled(...)`. Same for the calls in other test methods.